resources/views/objects/_usercard.blade.php

```diff
@@ -92,10 +94,10 @@ class="user-action-button user-action-button--message"
                     </div>
                 </div>
             </div>
-            <div class="usercard__status-bar usercard__status-bar--{{!isset($loading) && $user->isOnline() ? 'online' : 'offline'}}">
+            <div class="usercard__status-bar usercard__status-bar--{{ $showOnline ? 'online' : 'offline' }}">
                 <span class="far fa-fw fa-circle usercard__status-icon"></span>
-                <span class="usercard__status-message" title="{{isset($loading) || $user->isOnline() || $user->hide_presence ? '' : trans('users.show.lastvisit', ['date' => $user->user_lastvisit->diffForHumans()])}}">
-                    {{!isset($loading) && $user->isOnline() ? trans('users.status.online') : trans('users.status.offline')}}
+                <span class="usercard__status-message" title="{{ isset($loading) || is_null($user->displayed_last_visit) ? '' : trans('users.show.lastvisit', ['date' => $user->user_lastvisit->diffForHumans()]) }}">
```

Comment:
`$showOnline ? blah : ''`
Then it's uniform with the other parts of the usercard ?_?

Reply:
if they're online, it isn't supposed to show anything